sunpy/io/_file_tools.py

```diff
@@ -27,7 +31,8 @@
 _KNOWN_EXTENSIONS = {
     ('fts', 'fits'): 'fits',
     ('jp2', 'j2k', 'jpc', 'jpt'): 'jp2',
-    ('fz', 'f0'): 'ana'
+    ('fz', 'f0'): 'ana',
+    ('asdf', "asf"): 'asdf'
```

Comment:
```suggestion
    ('asdf', ): 'asdf'
```
Are there any known "asf" files?